### DISCUSSION TITLE
supported milvus version for Milvus CDC (Change Data Capture) tool

### DISCUSSION THREAD

**Discussion by rere950303** [milvus-io/milvus#33904] [Q&A and General discussion] [answered]
What is the milvus version that I can use milvus cdc tool? If i look at https://github.com/zilliztech/milvus-cdc/releases/tag/v1.0.0 , it says 2.4.0 or higher, and if i look at https://github.com/zilliztech/milvus-cdc/blob/main/doc/cdc-usage.md , it says 2.3.2 or higher, so I'm inquiring.
The milvus version I am currently using is 2.3.12.

**Comment by yhmo** [comment]
The cdc v1.0.0 was released 3 months ago. The last commit of cdc-usage.md was 7 months ago. So, I think the latest release version requires v2.4+.

**Reply by rere950303** [reply]
@yhmo https://milvus.io/docs/deploy-cdc-server.md#Prerequisites Looking here, I think the 2.3.12 version is also possible, but I'm confused, is it only possible for me to test it myself if it's also possible in 2.3.12?

@SimFG Can you check if cdc v1.0.0 is also available in milvus 2.3.12 version?

**Reply by SimFG** [reply, marked as answer]
@rere950303 Milvus 2.3.12 can be used, but some functions are not supported, but dml operations can be replicated.

**Reply by rere950303** [reply]
@SimFG thank you for replying!
![스크린샷 2024-06-17 오후 8 05 58](https://github.com/milvus-io/milvus/assets/78265252/41f605ab-adf6-4dbf-bc1a-6f63a464d237)
Does it mean that only `insert/delete/upsert` is supported in the picture above? Functions such as `create/drop collection` are also required.

**Reply by SimFG** [reply]
yes, insert/delete/upsert/ create/drop collection are supported when you use the cdc tool with the milvus 2.3.12

**Reply by rere950303** [reply]
@SimFG I'm sorry to mention it many times, but can you tell me which features are not supported in the milvus 2.3.12 version in the picture above?

**Comment by SimFG** [comment]
@rere950303 in the 2.3.12, insert/delete/upsert/ create/drop collection are supported, other apis are supported in the milvus 2.4.x

**Reply by rere950303** [reply]
thank you for replying!!
